src/app/Instance/BridgeOverview/BridgeOverview.stories.tsx

```diff
@@ -0,0 +1,12 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+import React from "react";
+import { BridgeOverview } from "./BridgeOverview";
+
+export default {
+  title: "Bridge/BridgeOverview",
```

Comment:
It's better to place this story inside the PoCs section of storybook.

```suggestion
  title: "PoCs/BridgeOverview",
```

Reply:
Okay.